apps/site/lib/site_web/views/schedule_view.ex

```diff
@@ -469,4 +469,22 @@ defmodule SiteWeb.ScheduleView do
       []
     end
   end
+
+  @spec timetable_note(Conn.t()) :: Safe.t()
+  def timetable_note(%{route: %Route{id: "CR-Fairmount"}}) do
```

Comment:
Should we avoid putting this note under the timetable for outbound? There is no FRANK cell for outbound.

Reply:
I wasn't sure. The ticket/design didn't indicate. I could do that, @ingridpierre / @katrinaml ?

Reply:
Makes sense to me - I'll take note of this since it's likely a bit brittle with rating changes.